Account_BLL.VerifyUserInfoIndex => User_BLL.GetUserInfo => LogRecord_Helper.RecordLog System.Data.SqlClient.SqlException (0x80131904): 用户 '' 登录失败。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PersonalWebService.DAL.OperateDB.GetOpenConnection() 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\OperateDB.cs:行号 113
   在 PersonalWebService.DAL.OperateDB.GetSingle[T](String sql, Object param) 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\OperateDB.cs:行号 50
   在 PersonalWebService.DAL.Operate_DAL.GetDataSingle[T](String sql, Object param) 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\Operate_DAL.cs:行号 37
   在 PersonalWebService.BLL.User_BLL.GetUserInfo(String email) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\User_BLL.cs:行号 115
ClientConnectionId:3eaa7498-5c9f-40b4-91d4-27f2633129eb
Error Number:18456,State:1,Class:14
SignController.LoginIndex => Account_BLL.VerifyUserInfoIndex => LogRecord_Helper.RecordLog NLog.NLogRuntimeException: Exception occurred in NLog ---> System.Data.SqlClient.SqlException: 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 26 - 定位指定的服务器/实例时出错)
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 NLog.Targets.DatabaseTarget.OpenConnection(String connectionString)
   在 NLog.Targets.DatabaseTarget.EnsureConnectionOpen(String connectionString)
   在 NLog.Targets.DatabaseTarget.WriteEventToDatabase(LogEventInfo logEvent)
   在 NLog.Targets.DatabaseTarget.Write(LogEventInfo logEvent)
   在 NLog.Targets.Target.Write(AsyncLogEventInfo logEvent)
   --- 内部异常堆栈跟踪的结尾 ---
   在 NLog.LoggerImpl.<>c__DisplayClass1.<Write>b__0(Exception ex)
   在 NLog.Internal.SingleCallContinuation.Function(Exception exception)
   在 NLog.Targets.Target.Write(AsyncLogEventInfo logEvent)
   在 NLog.Targets.Target.WriteAsyncLogEvent(AsyncLogEventInfo logEvent)
   在 NLog.LoggerImpl.WriteToTargetWithFilterChain(TargetWithFilterChain targetListHead, LogEventInfo logEvent, AsyncContinuation onException)
   在 NLog.LoggerImpl.Write(Type loggerType, TargetWithFilterChain targets, LogEventInfo logEvent, LogFactory factory)
   在 NLog.Logger.WriteToTargets(LogLevel level, IFormatProvider formatProvider, String message)
   在 NLog.Logger.Log(LogLevel level, String message)
   在 PersonalWebService.Helper.LogRecord_Helper.RecordLog(LogLevels loglevels, Object e) 位置 F:\Codes\PersonalWebSite\PersonalWebService.Helper\LogRecord_Helper.cs:行号 33
   在 PersonalWebService.BLL.User_BLL.GetUserInfo(String email) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\User_BLL.cs:行号 124
   在 PersonalWebService.BLL.Account_BLL.VerifyUserInfoIndex(UserLoginTwo user) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\Account_BLL.cs:行号 81
Account_BLL.VerifyUserInfoIndex => User_BLL.GetUserInfo => LogRecord_Helper.RecordLog System.Data.SqlClient.SqlException (0x80131904): 用户 '' 登录失败。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PersonalWebService.DAL.OperateDB.GetOpenConnection() 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\OperateDB.cs:行号 113
   在 PersonalWebService.DAL.OperateDB.GetSingle[T](String sql, Object param) 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\OperateDB.cs:行号 50
   在 PersonalWebService.DAL.Operate_DAL.GetDataSingle[T](String sql, Object param) 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\Operate_DAL.cs:行号 37
   在 PersonalWebService.BLL.User_BLL.GetUserInfo(String email) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\User_BLL.cs:行号 115
ClientConnectionId:f7c8d50c-cd6f-47eb-a5c4-c79dfc2e4bab
Error Number:18456,State:1,Class:14
SignController.LoginIndex => Account_BLL.VerifyUserInfoIndex => LogRecord_Helper.RecordLog NLog.NLogRuntimeException: Exception occurred in NLog ---> System.Data.SqlClient.SqlException: 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 26 - 定位指定的服务器/实例时出错)
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 NLog.Targets.DatabaseTarget.OpenConnection(String connectionString)
   在 NLog.Targets.DatabaseTarget.EnsureConnectionOpen(String connectionString)
   在 NLog.Targets.DatabaseTarget.WriteEventToDatabase(LogEventInfo logEvent)
   在 NLog.Targets.DatabaseTarget.Write(LogEventInfo logEvent)
   在 NLog.Targets.Target.Write(AsyncLogEventInfo logEvent)
   --- 内部异常堆栈跟踪的结尾 ---
   在 NLog.LoggerImpl.<>c__DisplayClass1.<Write>b__0(Exception ex)
   在 NLog.Internal.SingleCallContinuation.Function(Exception exception)
   在 NLog.Targets.Target.Write(AsyncLogEventInfo logEvent)
   在 NLog.Targets.Target.WriteAsyncLogEvent(AsyncLogEventInfo logEvent)
   在 NLog.LoggerImpl.WriteToTargetWithFilterChain(TargetWithFilterChain targetListHead, LogEventInfo logEvent, AsyncContinuation onException)
   在 NLog.LoggerImpl.Write(Type loggerType, TargetWithFilterChain targets, LogEventInfo logEvent, LogFactory factory)
   在 NLog.Logger.WriteToTargets(LogLevel level, IFormatProvider formatProvider, String message)
   在 NLog.Logger.Log(LogLevel level, String message)
   在 PersonalWebService.Helper.LogRecord_Helper.RecordLog(LogLevels loglevels, Object e) 位置 F:\Codes\PersonalWebSite\PersonalWebService.Helper\LogRecord_Helper.cs:行号 33
   在 PersonalWebService.BLL.User_BLL.GetUserInfo(String email) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\User_BLL.cs:行号 124
   在 PersonalWebService.BLL.Account_BLL.VerifyUserInfoIndex(UserLoginTwo user) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\Account_BLL.cs:行号 81
Account_BLL.VerifyUserInfoIndex => User_BLL.GetUserInfo => LogRecord_Helper.RecordLog System.Data.SqlClient.SqlException (0x80131904): 用户 '' 登录失败。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PersonalWebService.DAL.OperateDB.GetOpenConnection() 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\OperateDB.cs:行号 113
   在 PersonalWebService.DAL.OperateDB.GetSingle[T](String sql, Object param) 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\OperateDB.cs:行号 50
   在 PersonalWebService.DAL.Operate_DAL.GetDataSingle[T](String sql, Object param) 位置 F:\Codes\PersonalWebSite\PersonalWebService.DAL\Operate_DAL.cs:行号 37
   在 PersonalWebService.BLL.User_BLL.GetUserInfo(String email) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\User_BLL.cs:行号 115
ClientConnectionId:fdb86921-09ec-4f0c-bd84-baffae801e67
Error Number:18456,State:1,Class:14
SignController.LoginIndex => Account_BLL.VerifyUserInfoIndex => LogRecord_Helper.RecordLog NLog.NLogRuntimeException: Exception occurred in NLog ---> System.Data.SqlClient.SqlException: 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 26 - 定位指定的服务器/实例时出错)
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 NLog.Targets.DatabaseTarget.OpenConnection(String connectionString)
   在 NLog.Targets.DatabaseTarget.EnsureConnectionOpen(String connectionString)
   在 NLog.Targets.DatabaseTarget.WriteEventToDatabase(LogEventInfo logEvent)
   在 NLog.Targets.DatabaseTarget.Write(LogEventInfo logEvent)
   在 NLog.Targets.Target.Write(AsyncLogEventInfo logEvent)
   --- 内部异常堆栈跟踪的结尾 ---
   在 NLog.LoggerImpl.<>c__DisplayClass1.<Write>b__0(Exception ex)
   在 NLog.Internal.SingleCallContinuation.Function(Exception exception)
   在 NLog.Targets.Target.Write(AsyncLogEventInfo logEvent)
   在 NLog.Targets.Target.WriteAsyncLogEvent(AsyncLogEventInfo logEvent)
   在 NLog.LoggerImpl.WriteToTargetWithFilterChain(TargetWithFilterChain targetListHead, LogEventInfo logEvent, AsyncContinuation onException)
   在 NLog.LoggerImpl.Write(Type loggerType, TargetWithFilterChain targets, LogEventInfo logEvent, LogFactory factory)
   在 NLog.Logger.WriteToTargets(LogLevel level, IFormatProvider formatProvider, String message)
   在 NLog.Logger.Log(LogLevel level, String message)
   在 PersonalWebService.Helper.LogRecord_Helper.RecordLog(LogLevels loglevels, Object e) 位置 F:\Codes\PersonalWebSite\PersonalWebService.Helper\LogRecord_Helper.cs:行号 33
   在 PersonalWebService.BLL.User_BLL.GetUserInfo(String email) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\User_BLL.cs:行号 124
   在 PersonalWebService.BLL.Account_BLL.VerifyUserInfoIndex(UserLoginTwo user) 位置 F:\Codes\PersonalWebSite\PersonalWebService.BLL\Account_BLL.cs:行号 81
